src/script/engine.zig

```diff
@@ -524,6 +544,26 @@ test "Script execution - OP_RETURN" {
     }
 }
 
+test "Script execution - OP_TOALTSTACK OP_FROMALTSTACK" {
+    const allocator = std.testing.allocator;
+
+    // Simple script: OP_1 OP_TOALTSTACK OP_FROMALTSTACK
+    const script_bytes = [_]u8{
+        Opcode.OP_1.toBytes(),
+        Opcode.OP_TOALTSTACK.toBytes(),
+        Opcode.OP_FROMALTSTACK.toBytes(),
+    };
+    const script = Script.init(&script_bytes);
+
+    var engine = Engine.init(allocator, script, .{});
+    defer engine.deinit();
+
+    try engine.execute();
+
+    try std.testing.expectEqual(1, engine.stack.len());
+    try std.testing.expectEqual(0, engine.alt_stack.len());
```

Comment:
If both operand does nothing this test would pass too.
Try:
push1 push2 toalt toalt fromalt

The result must be stack[0] == 1, altstack[0] == 2

Reply:
@tdelabro it works 💯💯